python/frontend/cirrus/examples/logistic_regression.ipynb

```diff
@@ -184,6 +194,23 @@
     "        print(line)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test\n",
+    "---"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "test(server)"
```

Comment:
You should explain what this function does so the user know what's going on. Like we do in the other notebooks.